ci-operator/jobs/infra-periodics-origin-release-images.yaml

```diff
@@ -155,6 +155,32 @@ periodics:
         requests:
           cpu: 500m
     serviceAccountName: origin-release-images-builder
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci.openshift.io/role: infra
+  max_concurrency: 1
+  name: periodic-build-origin-release-image-origin-release-golang-1.20
+  spec:
+    containers:
+    - args:
+      - --namespace=ci
+      - start-build
+      - origin-release-golang-1.20
+      - --wait=true
+      command:
+      - oc
+      image: registry.ci.openshift.org/ocp/4.8:cli
```

Comment:
4.8?

Reply:
Yeah, I noticed that too. Cargo-culted from previous golang-* image additions. I also had a similar question about the `el7` repo being used...